Diagnostics/Get-NetView.PS1

```diff
@@ -776,6 +776,48 @@ function NativeNicDetail {
     }
 } # NativeNicDetail()
 
+function NicVendor {
+    [CmdletBinding()]
+    Param(
+        [parameter(Mandatory=$true)] [String] $OutDir
+    )
+
+    $dir = Join-Path $OutDir "NicVendor"
+    New-Item -Path $dir -ItemType Directory | Out-Null
+
+    foreach ($nic in Get-NetAdapter) {
+        $pciId = (Get-NetAdapterAdvancedProperty -Name $nic.Name -AllProperties -RegistryKeyword "ComponentID").RegistryValue
+        switch -Wildcard($pciId) {
+            # Not implemented
+            #"PCI\VEN_1425*" { ChelsioDetail $nic.ifIndex $dir }
```

Comment:
Meaning execute the call to a stub function for better clarity.  It's a NIT like I mentioned.